src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -331,19 +327,6 @@ private OperationStatus CheckAsyncMethodRefOutParameters(IList<VariableInfo> par
                 return OperationStatus.Succeeded;
             }
 
-            private static ValueTask<SemanticDocument> CreateDocumentWithAnnotationsAsync(SemanticDocument document, IList<VariableInfo> variables, CancellationToken cancellationToken)
-            {
-                var annotations = new List<Tuple<SyntaxToken, SyntaxAnnotation>>(variables.Count);
-                variables.Do(v => v.AddIdentifierTokenAnnotationPair(annotations, cancellationToken));
-
-                if (annotations.Count == 0)
-                {
-                    return ValueTaskFactory.FromResult(document);
-                }
-
-                return document.WithSyntaxRootAsync(document.Root.AddAnnotations(annotations), cancellationToken);
-            }
```

Comment:
moved to method-extractor which now does this *and* annotates the insertion point in one swell foop.